tsconfig.json

```diff
@@ -12,5 +11,11 @@
     { "path": "./packages/snaps-ui" },
     { "path": "./packages/snaps-utils" },
     { "path": "./packages/snaps-webpack-plugin" }
-  ]
+  ],
+  "compilerOptions": {
+    "esModuleInterop": true,
+    "noEmit": true
+  },
+  "files": [],
+  "include": ["scripts"]
```

Comment:
This is required to have proper type checking on the new `.ts` scripts in the `scripts` folder.